compiler/rustc_smir/src/rustc_internal/mod.rs

```diff
@@ -139,6 +139,10 @@ impl<'tcx> Tables<'tcx> {
         stable_mir::ty::Prov(self.create_alloc_id(aid))
     }
 
+    pub fn span(&mut self, span: Span) -> stable_mir::ty::Span {
+        self.create_span(span)
```

Comment:
This could be the body of a `Stable` impl, WO instead of removing the impl, put this logic into it.

Reply:
You can remove the `span` function now, it's not needed anymore